app/screens/channel/channel.tsx

```diff
@@ -136,6 +137,7 @@ const Channel = ({
                 mode='margin'
                 edges={edges}
                 testID='channel.screen'
+                onLayout={(e) => setContainerHeight(e.nativeEvent.layout.height)}
```

Comment:
I'm guessing this is not an issue with re-rendering?

Reply:
Made them like this to just try quickly, and forgot to refactor them. Will solve all.